### PR TITLE
v1.1.1: Update pipeline to nfcore 2.5.1

### PR DESCRIPTION
**Pull request by sguizard**
Update the pipeline to nf-core 2.5.1, update modules, and fix documentation.

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/isoseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/isoseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [X] Usage Documentation in `docs/usage.md` is updated.
- [X] Output Documentation in `docs/output.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,24 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.1 - White Hawk [26/09/2022]
```
Comment (maxulysse) [inline CHANGELOG.md:6]: released?

File: README.md [hunk 2]
```diff
@@ -31,7 +26,7 @@
...
 
-On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/isoseq/results).
+On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources.The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/isoseq/results).
```
Comment (maxulysse) [inline README.md:29]:
```suggestion
On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/isoseq/results).
```

Space should be there, I'll check template and fix if needed...

**Comment by maxulysse** [comment]
This is a PR from `dev` to `master`, so pipeline is already up to date with TEMPLATE, is this actually a release PR?

**Review by maxulysse — APPROVED** [review]
Looks good to me.
Only issue, this looks more like a release PR than an actual sync of TEMPLATE.
We do encourage all development to happen on `dev`, merge to `master` in a single PR that's reviewed by 2 people and then release.

**Comment by sguizard** [comment]
It's a PR release. 
I first merged the 2.5.1 updates from nf-core-template-merge-2.5.1 to dev. 
Then I updated modules and applied corrections on dev. 
Now I wish to release theses updates on master as version 1.1.1.

File: CHANGELOG.md (hunk 1 above)
Comment (sguizard) [reply CHANGELOG.md:6]: Yes, it's a release.

**Review by JoseEspinosa — APPROVED** [review]
LGTM!

**Comment by sguizard** [comment]
Thanks @maxulysse @JoseEspinosa !

**Pull request merged**